tgui/packages/tgui/interfaces/ERTManager.js

```diff
@@ -128,6 +129,15 @@ export const ERTManager = (props, context) => {
                 />
               ))}
             </LabeledList.Item>
+            <LabeledList.Item label="Security Module">
+            <Button
```

Comment:
```suggestion
              <Button
```
very minor nitpick, but since `button` is a child of the `labeledlist.item` it should be indented higher